### PR TITLE
Code style uncrustify config

### PR DESCRIPTION
**Pull request by igorkorsukov**
This is a config for `uncrustify` to format in a new style of code.
And the `.git-blame-ignore-revs` file to solve problems with `git blame`

See https://github.com/musescore/MuseScore/pull/6120

### CODE REVIEW

File: .git-blame-ignore-revs [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Conversion to new code style
+3626843dcac55d6f264a61251e745406159ea2e8
```
Comment (dmitrio95) [inline .git-blame-ignore-revs:2]: Shouldn't this be exactly the commit published in #6120? It looks like 3626843dcac55d6f264a61251e745406159ea2e8 won't be in git history of master branch.

File: tools/style/uncrustify_run_dir.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env sh
```
Comment (Jojo-Schmitz) [inline tools/style/uncrustify_run_dir.sh:1]: Good old plain `!/bin/sh` is POSIX, no trickery with /usr/bin/env is needed to find a bourne shell

File: .git-blame-ignore-revs (hunk 1 above)
Comment (igorkorsukov) [reply .git-blame-ignore-revs:2]: After the merger, 6020 will need to be checked and corrected.

File: tools/style/uncrustify_musescore.cfg [hunk 3]
```diff
@@ -0,0 +1,1615 @@
+# Uncrustify 0.71
```
Comment (Jojo-Schmitz) [inline tools/style/uncrustify_musescore.cfg:1]: Maybe eben `!/usr/bin/env uncrustify --no-backup -l CPP -c` ;-)

**Comment by Jojo-Schmitz** [comment]
my remarks here fixed in #6131

**Pull request merged**